### DISCUSSION TITLE
More verbose type hinting?

### DISCUSSION THREAD

**Discussion by fisher60** [encode/starlette#1596] [Potential Issue] [answered]
The return type hint at [this line of code](https://github.com/encode/starlette/blob/d7cbe2a4887ad6b15fe7523ed62e28a426b7697d/starlette/requests.py#L239) seems a bit vague, I recommend it is changed to `typing.Dict[str, typing.Any]` to accurately represent what this method is returning. I will open a corresponding PR since this is a very small change.

**Comment by Kludex** [comment, marked as answer]
https://github.com/python/typeshed/blob/32871c0d6468a3ee1eb2601a0e3f35ad7b5fb0b7/stdlib/json/__init__.pyi#L39-L49

**Reply by adriangb** [reply]
Yep json.loads will happily return an array or even scalar
